seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/TaskExecutionService.java

```diff
@@ -142,7 +142,7 @@ public TaskExecutionService(NodeEngineImpl nodeEngine, HazelcastProperties prope
         scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
         scheduledExecutorService.scheduleAtFixedRate(
                 this::updateMetricsContextInImap,
-                0,
+                30, // Wait for MapStore loading to complete, wait 30s
```

Comment:
You can check whether the MapStore has finished loading within the getJobMetricsBackupInterval() method. If it hasn't finished loading, just print a log and do not perform any other actions. This way, there is no need to add a delay at this point.

Reply:
> You can check whether the MapStore has finished loading within the getJobMetricsBackupInterval() method. If it hasn't finished loading, just print a log and do not perform any other actions. This way, there is no need to add a delay at this point.

Is there a specific method to check the loading status of MapStore, or is it determined by catching exceptions?

Reply:
if(NodeState.ACTIVE.equals(nodeEngine.getNode().getState())